packages/tests/src/services/ScreenshotService.ts

```diff
@@ -0,0 +1,72 @@
+import fs = require('fs');
+import SpectreClient from 'nodeclient-spectre';
+import { ScreenshotOutput } from 'webdriver-image-comparison/build/helpers/afterScreenshot.interfaces';
+
+// TODO - .env
+const SPECTRE_API_HOST= 'http://spectre.linkerd:4140';
+const PROJECT = 'Big Design';
+const SUITE = 'Components';
+
+export default class ScreenshotService {
+  runId: number
+  browser: WebdriverIO.BrowserObject
+  spectreClient: SpectreClient
```

Comment:
I believe can initialize and declare this in one line `spectreClient = new SpectreClient(SPECTRE_API_HOST)` without the need for a constructor

Reply:
✅